test/unit/mongo_client.test.js

```diff
@@ -44,23 +44,22 @@ describe('MongoOptions', function () {
      *
      * ### Additional options:
      *
-     * | nodejs native option  | driver spec compliant option name             | driver option type |
-     * |:----------------------|:----------------------------------------------|:-------------------|
-     * | `ca`                  | `tlsCAFile`                                   | `string`           |
-     * | `crl`                 | N/A                                           | `string`           |
-     * | `key`                 | `tlsCertificateKeyFile`                       | `string`           |
-     * | `passphrase`          | `tlsCertificateKeyFilePassword`               | `string`           |
-     * | `rejectUnauthorized`  | `tlsAllowInvalidCertificates`                 | `boolean`          |
-     * | `checkServerIdentity` | `tlsAllowInvalidHostnames`                    | `boolean`          |
-     * | see note below        | `tlsInsecure`                                 | `boolean`          |
+     * | nodejs native option    | driver spec compliant option name             | driver option type |
+     * |:------------------------|:----------------------------------------------|:-------------------|
+     * | `tlsCAFile`             | `tlsCAFile`                                   | `string`           |
+     * | `crl`                   | N/A                                           | `string`           |
+     * | `tlsCertificateKeyFile` | `tlsCertificateKeyFile`                       | `string`           |
```

Comment:
This shouldn't be changed, right?

Reply:
I may have misinterpreted what this table is for. I though the leftmost column corresponded to the option as it exists in node driver and the second column was the spec compliant option name. Is it actually the case that the leftmost column corresponds to node's _internal_ tls options?